src/Symfony/Bridge/Twig/Extension/CsrfRuntime.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bridge\Twig\Extension;
 
-use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+use Symfony\Component\Csrf\CsrfTokenManagerInterface;
```

Comment:
ALL similar changes are BC breaks

Reply:
Are you sure about this? There is a `class_alias` between the old and the new namespace.

I just tested this by linking the new CSRF component into the `vendor/` directory (+ autoload config) and relinking the old CSRF component into the `vendor/` directory. Then this code works as expected (without error + deprecation):

```php
<?php

require_once 'vendor/autoload.php';

use Symfony\Component\Security\Csrf\CsrfTokenManager;
use Symfony\Bridge\Twig\Extension\CsrfRuntime;

$manager = new CsrfTokenManager();
$runtime = new CsrfRuntime($manager);

echo $runtime->getCsrfToken('foo');
```